EIPS/eip-draft_grant_standard.md

```diff
@@ -26,69 +26,282 @@ The current process for grants in the Ethereum ecosystem is for membership organ
+
+The standard further delegates `voting` on payouts/refunds to the `manager` which can be an EOA or Contract (e.g. Multisig contract as `manager` or a DAO contract such as Moloch acting as `manager`). See discussion with [@NickSzabo4](https://twitter.com/NickSzabo4/) and [@ameensol](https://twitter.com/ameensol)
+
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">On-chain pools bring additional trust-minimization properties to cash distributions above doing a bunch of simple 1-to-1 payments. For example prevents payor from discriminating between payees: once sufficient funds are in the pool everybody gets their cash flow.</p>&mdash; Nick Szabo 🔑 (@NickSzabo4) <a href="https://twitter.com/NickSzabo4/status/1173281662718726145?ref_src=twsrc%5Etfw">September 15, 2019</a></blockquote> 
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">You&#39;re right, I should have been more general. The grants *standard* defines the minimum set of functions for pooling resources. Our reference implementation chose &quot;grantees can be cut off&quot;. But you could write the function your way and prevent additional categories of abuse. <a href="https://t.co/eNk6YmRPLH">pic.twitter.com/eNk6YmRPLH</a></p>&mdash; Ameen Soleimani 👹 (@ameensol) <a href="https://twitter.com/ameensol/status/1173317866801770496?ref_src=twsrc%5Etfw">September 15, 2019</a></blockquote>
+
+
+#### Why do we think the current interface is the best?
+
+The current interface supports managed and unmanaged grants (unmanaged being a completely trustless variety), various funding threshold and expiry schemes, and flexibility to award to multiple grantees and respond if one or more grantees misbehaves. The standard offers flexibility in implementation and a consistent interface.  
+
+#### Why do we think the parts we left out should be left out?
+
+The interface focused on requesting funds, funding, releasing / refunding funds. All other responsibilities are left out of the standard.
+
+Multiple grants are not handled within the standard and are instead managed using a factory contract. See the [reference implementation](https://github.com/NoahMarconi/grant-contracts/) for an example. 
+
+
 
 ## Backwards Compatibility
 <!--All EIPs that introduce backwards incompatibilities must include a section describing these incompatibilities and their severity. The EIP must explain how the author proposes to deal with these incompatibilities. EIP submissions without a sufficient backwards compatibility treatise may be rejected outright.-->
 
+N/A
 
 ## Test Cases
 <!--Test cases for an implementation are mandatory for EIPs that are affecting consensus changes. Other EIPs can choose to include links to test cases if applicable.-->
-TODO: write a sample contract with test cases
+
+[Reference Implementation](https://github.com/NoahMarconi/grant-contracts)
```

Comment:
Juicy! 🥇 

Do you have this deployed somewhere on testnet or mainnet? Worth linking to it if you do!

Reply:
Not deployed yet but can do that if we're good with the latest interface